### PR TITLE
document new rand.chacha function

### PR DESCRIPTION
**Pull request by moAlobaidi**
This PR closes this [issue](https://github.com/AleoHQ/techops/issues/48)

### CODE REVIEW

File: documentation/aleo/04_opcodes.md [hunk 1]
```diff
@@ -1225,6 +1226,40 @@ Raises `first` to the power of `second`, wrapping around at the boundary of the
...
 ***
 
+### `rand.chacha`
+
+#### Description 
+
+The `rand.chacha` opcode is used to generate random values within the `finalize` scope. It supports a wide range of types for the random value.
+#### Example Usage
+
+```rust
```
Comment (aharshbe) [inline documentation/aleo/04_opcodes.md:1236]: @moAlobaidi, change `rust` to `aleo` and you're good to go! Nice job.

**Review by aharshbe — CHANGES_REQUESTED** [review]
Hey @moAlobaidi I requested a simple change, [here](https://github.com/AleoHQ/welcome/pull/210/files#r1230213774). Basically for the code block, you need to change `rust` to `aleo` since these are Aleo Insutrctions. If you run it locally, you can see that it adds the syntax highlighting. I've already tested locally using your branch so no need to do so unless you're curious. You can just make the change using GitHub's UI 🙂

**Review by aharshbe — APPROVED** [review]
Added changes so approving.

**Comment by moAlobaidi** [comment]
@aharshbe Nice, thank you!

**Pull request merged**